### ISSUE TITLE
Sign up and Login Pages

### ISSUE THREAD

**Issue by hkasera** [hkasera/WeCare#6]
There should be a signup and a login page for users

**Comment by prachiagrawal269** [comment]
@hkasera aren't we supposed to create a "user" table in the backend with appropriate fields like "username", "passwordhash", "email" etc that will store the login credentials of registered users or we can simply move on with creating just the basic html pages for login and signup ?

**Comment by hkasera** [comment]
Yes, we do need a back end to this. As the project is in the initial stage, we do not have it yet. If you are confident, feel free to suggest back end schema along with the front end code. **Mongoose** is a good choice. :+1:

**Comment by nik-6947** [comment]
hey, as you asked we have created  a mock up of login and sign up pages. We have made 2 design structures, please comment on whichever is better or any changes required in either.
here is the link :
 https://moqups.com/Nishtha-3006/B710C8yD/p:a16b2c2a4
Also about the tools and framework we are going to use jQuery(to switch between login and signup) and Bootstrap or Materialize CSS. 
Thank You.
